server/server_test.go

```diff
@@ -153,7 +153,7 @@ func TestPublish(t *testing.T) {
 					Metadata: sampleValues,
 					Fields:   sampleValues,
 				},
-				expectedMsg: "timestamp: proto:\u00a0invalid nil Timestamp",
+				expectedMsg: "timestamp: proto: invalid nil Timestamp",
```

Comment:
This won't actually fix the problem, the error value is a deliberately and obviously random: https://github.com/elastic/elastic-agent-shipper/pull/90#discussion_r944688097

Reply:
Yea I'm not sure what's going on but without this change the CI failed on windows in the CI (and also on mac locally, though not in the CI?) -- so I'm not sure how to deal with this. Is there a way to make it work on all platforms?

Reply:
Merge/Rebase onto main, Denis fixed it here https://github.com/elastic/elastic-agent-shipper/pull/91/files#r944769758

Reply:
Ah, never mind I just saw the fix went in